src/state/zustand/swapperStore/actions.ts

```diff
@@ -187,3 +187,14 @@ export const updateTradeAmountsFromQuote = (set: SetSwapperStoreAction<SwapperSt
     false,
     `swapper/updateTradeAmountsFromQuote`,
   )
+
+export const setSwapperDefaultAffiliateBps = (set: SetSwapperStoreAction<SwapperState>) => () =>
```

Comment:
Not sure about this terminology here - if we're setting it, then it's not a `default` ? Default would be the initial state, any action that sets `affiliateBps` wouldn't set a default. Same thing for the selector name

Reply:
Actually the selector *does* return a default and *is* an initial state of sorts so the terminology is correct for it

Reply:
Actually looking into this implementation a bit more, do we even need an action at all? Wouldn't the selector effectively be enough? Since all this setter does is call the selector with the current state at the time of the call, and set `affiliateBps` as the return value of the selector call?

Reply:
We don't really need it, it is just an abstraction.

If removed it entirely in https://github.com/shapeshift/web/pull/4353/commits/b90024ea6dd4e11c94af84f00347767d56639bfd by coupling it with the swapper selection action, as that's where the logic should actually live.